NuGet.Config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <add key="templating" value="https://dotnet.myget.org/F/templating/api/v3/index.json" />
```

Comment:
According to @piotrpMSFT the automatic syncing with cli-deps has been set up but isn't working for an unknown reason. Ideally these packages will be available from cli-deps